parsers/rhea/rhea.go

```diff
@@ -0,0 +1,742 @@
+					Location:             description.Location.Resource}
+				rhea.Reactions = append(rhea.Reactions, newReaction)
+			case "http://rdf.rhea-db.org/BidirectionalReaction":
+				newReaction := Reaction{
+					ID:                   description.ID,
+					Directional:          false,
+					Accession:            description.Accession,
+					Status:               description.Status.Resource,
+					Comment:              description.Comment,
+					Equation:             description.Equation,
+					HTMLEquation:         description.HTMLEquation,
+					IsChemicallyBalanced: description.IsChemicallyBalanced,
+					IsTransport:          description.IsTransport,
+					Ec:                   description.EC.Resource,
+					Citations:            description.CitationStrings(),
+					Substrates:           description.SubstrateStrings(),
+					Products:             description.ProductStrings(),
+					SubstrateOrProducts:  description.SubstrateOrProductStrings(),
+					Location:             description.Location.Resource}
+				rhea.Reactions = append(rhea.Reactions, newReaction)
+			case "http://rdf.rhea-db.org/SmallMolecule", "http://rdf.rhea-db.org/Polymer":
+				compoundType := subclass.Resource[23:]
+				newCompound := Compound{
+					ID:        description.ID,
+					Accession: description.About,
+					Position:  description.Position,
+					Name:      description.Name,
+					HTMLName:  description.HTMLName,
+					Formula:   description.Formula,
+					Charge:    description.Charge,
+					Chebi:     description.Chebi.Resource,
+
+					CompoundID:        description.ID,
+					CompoundAccession: description.Accession,
+					CompoundName:      description.Name,
+					CompoundHTMLName:  description.HTMLName,
+					CompoundType:      compoundType}
+				if compoundType == "Polymer" {
+					newCompound.Chebi = description.UnderlyingChebi.Resource
+				}
+				// Add subclass Chebi
+				for _, sc := range description.Subclass {
+					if strings.Contains(sc.Resource, "CHEBI") {
+						newCompound.SubclassOfChebi = sc.Resource
+					}
+				}
+				// Add new reactive parts and new compounds to rhea
+				rhea.Compounds = append(rhea.Compounds, newCompound)
+			case "http://rdf.rhea-db.org/GenericPolypeptide", "http://rdf.rhea-db.org/GenericPolynucleotide", "http://rdf.rhea-db.org/GenericHeteropolysaccharide":
+				compoundType := subclass.Resource[23:]
+				newCompound := Compound{
+					Accession:        description.About,
+					CompoundID:       description.ID,
+					CompoundName:     description.Name,
+					CompoundHTMLName: description.HTMLName,
+					CompoundType:     compoundType}
+				compoundMap[description.About] = newCompound
+				compoundParticipantMap[description.ReactivePartXML.Resource] = description.About
+			}
+		}
+	}
+
+	// Go back and get the ReactiveParts
+	for _, description := range rdf.Descriptions {
+		for _, subclass := range description.Subclass {
+			switch subclass.Resource {
+			case "http://rdf.rhea-db.org/ReactivePart":
```

Comment:
```suggestion
			if subclass.Resource == "http://rdf.rhea-db.org/ReactivePart":
```
There is only one case.

Also factoring out this code block would increase readability and you can unit test it.

Reply:
Got it, changing